README.md

```diff
@@ -83,13 +83,13 @@ And to deploy a master node with the coreos image (image\_source above) and pass
 ```
 mkdir -p configdrive/openstack/latest
 cp ocp/master.ign configdrive/openstack/latest/user_data
-openstack baremetal node deploy --config-drive configdrive $NODE_UUID
+openstack baremetal node deploy --config-drive configdrive $NODE_UUID --wait
 ```
 
 To ssh to the master nodes, you can route trafic through the bootstrap node
 ```
 sudo ip route add 172.22.0.0/24 via $(getent hosts ostest-api.test.metalkube.org | awk '{ print $1 }')
```

Comment:
$ getent hosts ostest-api.test.metalkube.org 
Is now returning 4 entries, which breaks this command.

Reply:
Fixed